### PR TITLE
[Controller] Fix PostResponseEventon invokeAction if the response is not return

### PR DESCRIPTION
**Pull request by fkroockmann**
https://github.com/backbee/backbee-standard/issues/306

### CODE REVIEW

File: Controller/FrontController.php [hunk 1]
```diff
@@ -563,9 +562,12 @@ protected function invokeAction($type = self::MASTER_REQUEST)
 
             // get controller action arguments
             $actionArguments = $controllerResolver->getArguments($request, $controller);
-
+            
             $response = call_user_func_array($controller, $actionArguments);
-            $dispatcher->dispatch($eventName.'.postcall', new Event\PostResponseEvent($response, $request));
+            
+            if (null !== $response && null !== $request) {
```
Comment (eric-chau) [inline Controller/FrontController.php]: to be really strict you should do this test:
```php
if ($response instanceof Response && $request instanceof Request) {
```
Keep in mind that $response can be a string too.

**Comment by eric-chau** [comment]
btw this is a duplication of #477 made by @mickaelandrieu.

File: Controller/FrontController.php [hunk 2]
```diff
@@ -310,10 +310,9 @@ public function defaultAction($uri = null, $sendResponse = true)
                 );
 
                 $this->send($response);
-                $this->application->stop();
             }
         }
-
+
```
Comment (ndufreche) [inline Controller/FrontController.php]: trim whitespaces

File: Controller/FrontController.php [hunk 3]
```diff
@@ -538,7 +537,7 @@ protected function invokeAction($type = self::MASTER_REQUEST)
         $request = $this->getRequest();
         $controllerResolver = $this->getApplication()->getContainer()->get('controller_resolver');
         $controller = $controllerResolver->getController($request);
-
+
```
Comment (ndufreche) [inline Controller/FrontController.php]: trim whitespaces

File: Controller/FrontController.php [hunk 4]
```diff
@@ -563,9 +562,12 @@ protected function invokeAction($type = self::MASTER_REQUEST)
 
             // get controller action arguments
             $actionArguments = $controllerResolver->getArguments($request, $controller);
-
+
```
Comment (ndufreche) [inline Controller/FrontController.php]: trim whitespaces

**Comment by fkroockmann** [comment]
Ticket was assigned to me, i don't see this PR :/

**Comment by eric-chau** [comment]
I wasn't blaming you at all ;)

**Pull request closed without merging**